js/settings.js

```diff
@@ -171,6 +171,17 @@ function av_mode_show_options(str){
 	$('#av_host, #av_port, #av_request_service, #av_response_header').attr('disabled', false);
 	$('p.av_host, p.av_port, p.av_mode_icap').show('slow');
 }
+
+function infected_action_show_options(str){
+	if (str === 'true') {
+		$('p.infected_action').show('slow');
+		$('#av_infected_action').attr('disabled', false);
```

Comment:
Disabling might not be the best option. The option will be shown or hidden along with the `p.infected_action`, so I don't think we need to do anything.
This will leave us with fewer lines of code, which can be moved inside the callback, no need to create a function for now